cluster-provision/k8s/1.20/provision.sh

```diff
@@ -56,56 +66,29 @@ yum -y remove firewalld
 # Required for iscsi demo to work.
 yum -y install iscsi-initiator-utils
 
-# To prevent preflight issue realted to tc not found
+# To prevent preflight issue related to tc not found
 dnf install -y tc
 
-# Install docker required packages.
-dnf -y install yum-utils \
-    device-mapper-persistent-data \
-    lvm2
-
-# Add Docker repository.
-dnf config-manager --add-repo=https://download.docker.com/linux/centos/docker-ce.repo
+export CRIO_VERSION=1.20
+curl -L -o /etc/yum.repos.d/rhrazdil:devel:kubic:libcontainers:stable.repo https://rhrazdil.github.io/crio-mirror/rhrazdil_devel_kubic_libcontainers_stable.repo
```

Comment:
Hello @rmohr, I have created a crio repository mirror on my github, the tests are looking good.
How do we proceed further? Will we create this repo under our kubevirt org?